documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -145,12 +145,14 @@ times can be configured by adding the
 link:http://docs.jboss.org/infinispan/{infinispanversion}/configdocs/infinispan-config-{infinispanversion}.html[&lt;expiration /&gt;]
 element to your `<*-cache />`  configuration sections.
 
-When an entry expires it will reside in the data container or cache store
-until it is accessed again by a user request.  There is also an optional
-expiration reaper that can run at a given configurable interval of
-milliseconds which will check for expired entries and remove them. Expiration reaper
-is always enabled when a cache store is present. It is highly recommended to always enable
-the reaper when using max idle in a clustered cache.
+When an entry expires it resides in the data container or cache store
+until it is accessed again by a user request. A reaper is also available to check for expired entries and remove them. The reaper runs at a configurable interval of milliseconds and is always enabled when a cache store is present.
```

Comment:
The more I wonder, maybe this should be stated as "cannot be disabled" instead of "is always enabled". The reaper is enabled by default as well. Not sure if we should mention that here as well.